frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -1,106 +1,164 @@
-.defence-mechanism {
-	cursor: default;
+.defence-mechanism-fieldset {
+	min-height: fit-content;
+	margin: 0.1em;
+	border: 0.25rem solid var(--control-config-border);
+	border-radius: 0.5rem;
+}
+
+/* the name of the fieldset */
+.defence-mechanism-legend {
+	padding: 0.3em;
 }
 
-.defence-mechanism > * {
-	padding: 0.75rem;
+/* the more button to expand the details */
```

Comment:
I'm happy to remove the comments for the different classes, I thought it would help as folks review and then someone would tell me to remove the comments 😄

Reply:
@dhinrichs-scottlogic that's really thoughtful! Was pretty helpful actually for review :)